src/core/providers/ogr/qgsogrprovider.cpp

```diff
@@ -4418,6 +4418,9 @@ void QgsOgrProvider::open( OpenMode mode )
 
     QgsDebugMsg( "OGR opened using Driver " + mGDALDriverName );
 
+    if ( mGDALDriverName == "GPKG" )
+      QgsGeoPackageProviderConnection::enableAmphibiousMode( mFilePath.split( '|' ).at( 0 ) );
```

Comment:
I don't see the need for this. This is already done in the OGR GPKG driver per https://github.com/OSGeo/gdal/blob/5870c3fc8ccc05ead6b9c8a5dd9fad76969fd890/gdal/ogr/ogrsf_frmts/gpkg/ogrgeopackagedatasource.cpp#L6732

Reply:
Ah great! I just thought it wasn't enabled anywhere, I'll just remove the c++ code, leave the test and the python code. Though when I made a trigger with db manager it would simply break in qgis, and it thought this might solve this.

Reply:
Welp it works fine without, maybe its simply that between the dbmanager trigger and the use in QGIS the trigger changed or got removed.